sequential/backend/seq_to_pat.pyx

```diff
@@ -21,11 +21,14 @@ from libcpp.vector cimport vector
 # then each setter is called via settattr() and
 # assigned to its corresponding c_seq2pat element
 cdef class PySeq2pat:
-    cdef Seq2pat c_seq2pat
+    cdef Seq2pat* c_seq2pat;
```

Comment:
why a pointer now?

Reply:
The idea here can be referring to a cython doc [page](https://cython.readthedocs.io/en/stable/src/userguide/wrapping_CPlusPlus.html), introducing the wrapper class. I am trying to define it as a pointer and explicitly delete this pointer in the `__dealloc__` method to deallocate the memory.